spec/features/welcome_spec.rb

```diff
@@ -68,4 +68,25 @@
     expect(page).to have_current_path(root_path)
   end
 
+  scenario 'is not shown to a regular user if the verification its overrided and the user has already logged in 1 time' do
+    user = create(:user, sign_in_count: 1)
+
+    Setting["feature.user.skip_verification"] = 'true'
```

Comment:
Remember every time a spec alters a setting, it should revert the change back afterwards, flakys on random spec order are born because of this